### PR TITLE
libcapn 2.0.0

### PR DESCRIPTION
**Pull request by ilovezfs**
also some style tweaks

### CODE REVIEW

File: Formula/libcapn.rb [hunk 1]
```diff
@@ -20,58 +20,32 @@ class Libcapn < Formula
...
-    spec = Tab.for_name("libcapn").source["spec"]
-    example = if spec == "stable"
-      "send_push.c"
-    elsif spec == "head" || spec == "devel"
-      "send_push_message.c"
-    end
-    cp "#{doc}/examples/#{example}", testpath
+    cp pkgshare/"examples/send_push_message.c", testpath
     flags = [
       "-I#{Formula["openssl"].opt_prefix}/include",
```
Comment (DomT4) [inline Formula/libcapn.rb]: Not your fault, but this should be `opt_include` rather than `opt_prefix/include`.

File: Formula/libcapn.rb [hunk 2]
```diff
@@ -20,58 +20,32 @@ class Libcapn < Formula
...
-      url "https://github.com/akheron/jansson.git", :revision => "e44b2231b50aea5de78b7ea2debec0d5327cd711"
-    end
-  end
-
   depends_on "cmake" => :build
   depends_on "pkg-config" => :build
   depends_on "openssl"
 
   def install
     if build.stable?
```
Comment (DomT4) [inline Formula/libcapn.rb]: Do we still need to check this with `if build.stable?`

Comment (ilovezfs) [inline Formula/libcapn.rb]: Yes. jansson is actually a submodule, but upstream hasn't set a specific revision. For stable, I arbitrarily pegged it to the current HEAD revision of jansson, but we could let it float for both stable and head and drop the resource block, if that's preferred.

Comment (ilovezfs) [inline Formula/libcapn.rb]: @DomT4 do we have a general policy on git submodules that point to their respective repository's head in a stable spec?

Comment (DomT4) [inline Formula/libcapn.rb]: Not a hard one, no. It makes sense to peg it to a specific commit for stable/devel & roam around freely for `head`, IMO. This _seems_ reasonable to me as-is.

File: Formula/libcapn.rb (hunk 1 above)
Comment (ilovezfs) [inline Formula/libcapn.rb]: Audit checks for that would be nice, like the pkgshare one

**Comment by ilovezfs** [comment]
@DomT4 PR refreshed.

File: Formula/libcapn.rb [hunk 3]
```diff
@@ -20,58 +20,28 @@ class Libcapn < Formula
...
-      ENV.cflags.to_s.split,
-    ].flatten
-    system ENV.cc, "-o", "example", example, *flags
-    if spec == "stable"
-      assert_match /unable to use specified SSL certificate \(12\)/, shell_output("./example", 1)
-    elsif spec == "head" || spec == "devel"
-      assert_match /unable to use specified PKCS12 file \(errno: 9012\)/, shell_output("./example", 255)
-    end
+    cp pkgshare/"examples/send_push_message.c", testpath
+    system ENV.cc, pkgshare/"examples/send_push_message.c",
```
Comment (DomT4) [inline Formula/libcapn.rb]: Should this be `testpath/"send_push_message.c"`? Bit confused that it was copied to the `testpath` & then seems to be unused, but may be missing something obvious.

Comment (DomT4) [inline Formula/libcapn.rb]: 👍 on deleting that line.

**Pull request closed without merging**